database/seeds/4-grants.js

```diff
@@ -0,0 +1,202 @@
+exports.seed = function(knex) {
+  // Deletes ALL existing entries
+  return knex("table_name")
+    .del()
+    .then(function() {
+      // Inserts seed entries
+      return knex("grants").insert([
+        {
+          user_id: user,
+          grant_title: "FedEx Small Business Grant",
+          grant_number: Math.round(Math.random()),
```

Comment:
Where is this user coming from? Are you assigning all these grants to the same user?